django/db/models/expressions.py

```diff
@@ -953,3 +953,25 @@ def asc(self):
 
     def desc(self):
         self.descending = True
+
+
+class Cast(Func):
+    """Coerce a lookup to a new field type."""
+    mysql_types = {
+        fields.CharField: 'char',
+        fields.IntegerField: 'signed integer',
+        fields.FloatField: 'signed',
+    }
+
+    def __init__(self, lookup, output_field):
```

Comment:
`lookup` isn't a good name for this argument - since it can be any expression, I would call it `expression`